content_sync/apis/github.py

```diff
@@ -159,23 +171,25 @@ def upsert_content_files_for_user(self, user_id=None) -> Commit:
         unsynced_states = ContentSyncState.objects.filter(
             Q(content__website=self.website) & Q(content__updated_by=user_id)
         ).exclude(
-            Q(current_checksum=F("synced_checksum")) & Q(synced_checksum__isnull=False)
+            Q(current_checksum=F("synced_checksum"))
```

Comment:
I noticed that if I log in as a particular user and delete one file, `upsert_content_files_for_user` is never called for that user, unless there are some other unsynced modified content objects.  This seems to be because the query used in `upsert_content_files` above (`website.websitecontent_set`) does not pick up the deleted content.

Reply:
Huh, I swear I wrote a test to cover this, i'll double check